  File "app.py", line 14
SyntaxError: Non-ASCII character '\xe5' in file app.py on line 14, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
